### PR TITLE
Add github action to update install script and crd files

### PR DESCRIPTION
**Pull request by cwen0**
Signed-off-by: cwen0 <cwenyin0@gmail.com>

### What problem does this PR solve?
 

Add two GitHub action to update install script and CRD files to AWS s3 service. 

### Check List

Tests

- Unit test
- E2E test
- Manual test (add detailed scripts or steps below)
- No code

Code changes

- Has Go code change
- Has CI related scripts change
- Has Terraform scripts change

Side effects

- Breaking backward compatibility

Related changes

- Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/update_crd.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: Update chaos mesh crd
```
Comment (g1eny0ung) [inline .github/workflows/update_crd.yml:1]: What about **Upload**?

Comment (cwen0) [reply .github/workflows/update_crd.yml:1]: Got

File: .github/workflows/update_crd.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - manifests/crd.yaml
+      - manifests/chaosfs-sidecar.yaml
+jobs:
+  run:
+    name: Upload media files
```
Comment (g1eny0ung) [inline .github/workflows/update_crd.yml:11]: This name also needs to be changed.

File: .github/workflows/uoload_crd.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+          pip3 install --upgrade setuptools
+          pip3 install awscli
+          printf "%s\n" ${{ secrets.AWS_ACCESS_KEY }} ${{ secrets.AWS_SECRET_KEY }} ${{ secrets.AWS_REGION }} "json" | aws configure
+      - name: Upload crd file
+        run: |
+          GIT_TAG=${GITHUB_REF##*/}
+          if [[ "$GIT_TAG" == "master" ]]; then
+            GIT_TAG="latest"
+          fi
+          cloud-assets-utils verify-and-sync manifests/crd.yaml -aws true -aws-bucket ${{ secrets.AWS_BUCKET_NAME }} -replace-first-path-to ${GIT_TAG}/crd.yaml
```
Comment (g1eny0ung) [inline .github/workflows/uoload_crd.yml:36]: Should be `${GIT_TAG}/manifests/crd.yaml`.

Comment (g1eny0ung) [reply .github/workflows/uoload_crd.yml:36]: The filename also has a typo 🤣, should be `upload_crd.yml`.

File: .github/workflows/upload_crd.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      - name: Download cloud_assets_utils
+        run: |
+          curl -L https://github.com/pingcap/cloud-assets-utils/releases/download/v0.2.0/cloud_assets_utils-ubuntu-latest -o cloud_assets_utils
+          chmod +x cloud_assets_utils
+          sudo mv cloud_assets_utils /usr/local/bin/cloud-assets-utils
+      - name: Configure awscli
+        run: |
+          pip3 install --upgrade setuptools
+          pip3 install awscli
+          printf "%s\n" ${{ secrets.AWS_ACCESS_KEY }} ${{ secrets.AWS_SECRET_KEY }} ${{ secrets.AWS_REGION }} "json" | aws configure
```
Comment (zhouqiang-cl) [inline .github/workflows/upload_crd.yml:29]: Did the key will print out?

**Pull request merged**